docs/_diagrams/uml/sch-iso.uml

```diff
@@ -0,0 +1,48 @@
+@startuml
+
+skinparam backgroundColor transparent
+skinparam arrowThickness 2
+skinparam fixCircleLabelOverlapping true
+
+skinparam database {
+  fontColor #f1f1f1
+  backgroundColor #2f6190
+  borderThickness 2
+}
+skinparam rectangle {
+  borderColor<<schema>> transparent
+  fontColor<<schema>> #f1f1f1
+  stereotypeFontColor<<schema>> transparent
+  backgroundColor<<schema>> transparent
+  borderColor #181818
+  fontColor #181818
+  backgroundColor #f1f1f1
+  borderThickness 2
+}
+skinparam interface {
+  borderThickness 2
+}
+
+database "PostgreSQL DB" {
+  rectangle internal <<schema>> {
+    rectangle tables_internal as "tables"
+  }
+  rectangle private <<schema>> {
+    rectangle tables_private as "tables"
+  }
+  rectangle core <<schema>> {
+    rectangle tables_core as "tables"
+  }
```

Comment:
How about just having `public` and `extensions` as schemas.

I recall a user reported internal, core, private was confusing.